src/components/MemberForm.js

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from 'semantic-ui-react';
+
+export default function MemberForm({ members, setMembers }) {
+  const [inputNames, setInputNames] = useState('');
+  const [inputWarning, setInputWarning] = useState('');
+
+  const handleMemberSubmit = event => {
+    event.preventDefault();
+    const memberArray = inputNames
+      .split(',')
+      .map(function (name) {
+        return name.trim();
+      })
+      .filter(name => {
+        return name !== '';
+      });
+    console.log(memberArray);
+    if (memberArray.length > 0) {
+      setMembers(memberArray);
+      setInputWarning('');
+    } else {
+      setMembers([]);
+      setInputWarning('Invalid input');
```

Comment:
Do you really need another state for the input warning? In other words: Is the warning depending on another state maybe?